docs/91_version_history.rst

```diff
@@ -35,229 +35,186 @@ Release Date: 2023-10-17
-       - The default interval for running the purge process has been reduced, the purge process will not run every 5 minutes. This chance only affects new installations, the purge process will run as before on systems that are upgraded.
-       - A change has been made to prevent the schedules used to start services from being renamed as this could cause the services to fail.
-       - The table registration mechanism has been extended to allow services to register for delete operations.
-       - The public API has been updated to include the ability to make control requests via the public API.
        - The support bundle creation process has been updated to include any performance counters available in the system.
        - The ability to monitor performance counters has been added to Fledge. Currently only south services offer performance counters that can be captured by the system. These are designed to provide information useful for tuning the south service.
+       - The asset tracker component has been optimized in order to improve the ingress and egress performance of Fledge
+       - The process used to extract log information from the system logs has been updated to improve performance and reduce the system overhead required to extract log data.
+       - The mechanism used by the south and north services to interact with the audit log has been optimized. This improves the ingress and egress performance of the product at the cost of a small delay before the audit log is updated.
+       - A number of changes have been made to improve the performance of sending data north from the system.
+       - The performance of the statistics history task has been improved. It now makes fewer calls to the storage subsystem, improving the overall system performance.
+       - The performance of the asset tracker system has been improved, resulting in an improvement in the ingress performance of the system.
+       - A number of optimizations have been made to improve the performance of Python filters within a pipeline.
+       - The SQLite In-Memory storage plugin now has an option that allows the data to be persisted when shutting the system down and reloaded on startup.
+       - A number of optimisations to the SQLite in-memory storage plugin and the SQLiteLB storage plugin have been added that increase the rate at which readings can be stored with these plugins.
+       - Changes have been made to the purge process in the sqlitelb and sqlite-in-memory plugins in order to improve performance.
+       - The SQLite storage plugins have been updated to improve the error reporting around database contention issues.
+       - The default interval for running the purge process has been reduced, the purge process will not run every 5 minutes. This change only affects new installations, the purge process will run as before on systems that are upgraded.       
```

Comment:
Both of these sentences are too long. They would be better if you started new sentences where the commas are now. I don't understand the meaning of "the purge process will not run every 5 minutes." Would it not be better to document the frequency the purge process _**does**_ run at?